src/index.ts

```diff
@@ -39,8 +39,6 @@ export default function (
 
       return {
         headTags: [`
-        <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
```

Comment:
Forgot I removed this as I already had it in the wrapping site, it should be an option I guess.